riscv/src/lib.rs

```diff
@@ -36,6 +36,7 @@
 #![allow(clippy::missing_safety_doc)]
 
 pub mod asm;
+pub(crate) mod bits;
```

Comment:
we can leave it as this, but now I don't find any reason why we should hide it from dependents. Maybe PACs find them handy

Reply:
I'd hold off on that until we're confident it's a stable public api.